helios-system-tests/src/main/java/com/spotify/helios/system/MasterAuthenticationTest.java

```diff
@@ -76,8 +92,8 @@ public String schemeName() {
     }
 
     @Override
-    public ServerAuthentication<String> serverAuthentication() {
-      return new ServerAuthentication<String>() {
+    public ServerAuthentication<String> serverAuthentication(Map<String, String> environment) {
+      return new SimpleServerAuthentication<String>() {
         @Override
         public Authenticator<String> authenticator() {
           return new Authenticator<String>() {
```

Comment:
I tried to stage this into separate commits but in reality i wrote all the code at once; probably introduced it then